docs/docs/configuration.md

```diff
@@ -125,13 +125,15 @@ export function updateGravatar(
 Once the `uncrashable-config.yaml` has been created run the following:
 
 ```sh
-graph codegen
+graph codegen -u
 ```
 
 and then run the graph uncrashable codegen.
 
 Now you can import the generated functions from the `UncrashableHelpers.ts` into your subgraph mapping files.
 
+Additionally the `-uc <path-to-config>` flag can be used to set the config file used by uncrashable. This is useful if you have multiple configurations in the same codebase. Along the same lines the output folder can be renamed using the `-o <folder-name>` command.
```

Comment:
The `-uc` command currently breaks. Not sure what the issue is. But we'll need it in our codebase.